st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -238,6 +253,51 @@ def run(self, action_parameters):
 
         return (status, partial_results, exec_context)
 
+    def resume(self, ex_ref, task_names):
+        mistral_ctx = ex_ref.context.get('mistral', dict())
+
+        if not mistral_ctx.get('execution_id'):
+            raise Exception('Unable to rerun because mistral execution_id is missing.')
+
+        execution = self._client.executions.get(mistral_ctx.get('execution_id'))
+
+        # pylint: disable=no-member
+        if execution.state not in ['ERROR']:
```

Comment:
I was confused for a second, but not I see this refers to the Mistral execution and not StackStorm one :)